met-web/src/components/engagement/form/EngagementFormTabs/EngagementForm.tsx

```diff
@@ -180,19 +180,19 @@ const EngagementForm = () => {
                 direction="row"
                 justifyContent="flex-start"
                 alignItems="flex-start"
-                rowSpacing={2}
+                rowSpacing={4}
                 sx={{ padding: '2em' }}
             >
                 <DayCalculatorModal open={isOpen} updateModal={setIsOpen} />
                 <Grid item xs={12} lg={12} md={12}>
-                    <MetLabel sx={{ marginBottom: '2px' }}>Engagement Name </MetLabel>
+                    <MetLabel>Engagement Name </MetLabel>
                     <FormLabel
-                        id="header"
+                        id="name"
                         sx={{
-                            mb: 1,
+                            fontSize: '13px',
```

Comment:
I think we are trying to not inject font size changes like this. We create our component in common with the needed size then we reuse it.